bikeshed/__init__.py

```diff
@@ -2551,8 +2558,15 @@ def inlineRemoteIssues(doc):
                 if el.tag == "p":
                     el.tag = "div"
                 addClass(el, "no-marker")
-        except:
-            pass
+        except urllib2.HTTPError as err:
+            if doc.token and err.code == 401:
+                die("Unauthorized Access to GitHub's API. There might be an issue with your token.")
+            elif doc.token is None and err.code == 403:
+                die("You've reached GitHub API's rate limit. Please provide an auth token.")
+            else:
+                die("{0}", err)
```

Comment:
Here and the next die(), please supply an intro message, then stringify the error message onto the next line.  Like, `"Error inlining GitHub issues:\n{0}"`.